script/list-of-files.txt

```diff
@@ -1,3 +1,4 @@
 ./README.md
 ./_site/CONTRIBUTING.md
 ./_site/index.html
+find . -type f -name "*.html"
```

Comment:
```sh
find . -name '*.html'
```

@brianamarie, should be enough, it's always a file with an extension.

> Find files by extension:
> `find {{root_path}} -name '{{*.ext}}'`

via https://github.com/tldr-pages/tldr/blob/master/pages/common/find.md

---

https://github.com/tldr-pages/tldr is my favourite CLI tools to quickly look up these things